src/imageops/mod.rs

```diff
@@ -164,6 +164,21 @@ where
     }
 }
 
+/// Tile an image by repeating it multiple times
```

Comment:
Could you add the example here? Marked with `no_run` since it tries to load from disk.